system-config/usbguard.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+DefaultDependencies=no
+ConditionKernelCommandLine=|!rd.qubes.hide_all_usb
+ConditionKernelCommandLine=|rd.qubes.hide_all_usb=0
+ConditionKernelCommandLine=|rd.qubes.hide_all_usb=no
+ConditionKernelCommandLine=|rd.qubes.hide_all_usb=off
+ConditionKernelCommandLine=|rd.qubes.dom0_usb
+ConditionKernelCommandLine=usbcore.authorized_default=0
+ConditionKernelCommandLine=!usbcore.nousb
+Before=systemd-ask-password-plymouth.path systemd-ask-password-wall.path
+
+[Service]
+ExecStart=
+ExecStart=/usr/sbin/usbguard-daemon -f -s -c /etc/usbguard/qubes-usbguard.conf
+
+[Install]
+WantedBy=debug-shell.service rescue.target sysinit.target emergency.target
```

Comment:
`rescue.target` already pulls in `sysinit.target`, so no need to list the former. Also I believe `debug-shell.service` shouldn't be here (it's just enabled by relevant option on kernel cmdline, but normal targets are pulled in regardless).